src/Oci8/Schema/Grammars/OracleGrammar.php

```diff
@@ -158,7 +158,7 @@ protected function addPrimaryKeys(Blueprint $blueprint)
      */
     public function compileTableExists()
     {
-        return "select * from user_tables where upper(table_name) = upper(?)";
+        return "select * from all_tables where upper(table_name) = upper(?)";
```

Comment:
I think if we want to use `all_tables`, we also need to insert the schema/owner in the query:
```php
select * from all_tables where upper(table_name) = upper(?) and upper(owner) = upper(?)
```

But the thing is how do we past the schema name on the binding? Not yet sure how to do it. Will try to dig this further later. Thanks!